gtests/net/mptcp/dss/dss_fin_data_no_mpc_fallback.pkt

```diff
@@ -0,0 +1,27 @@
+// Test data_fin retransmission from kernel TCP_ESTABLISHED state
+--tolerance_usecs=100000
+`../common/defaults.sh`
+
++0     socket(..., SOCK_STREAM, IPPROTO_MPTCP) = 3
++0     setsockopt(3, SOL_SOCKET, SO_REUSEADDR, [1], 4) = 0
+
++0     bind(3, ..., ...) = 0
++0     listen(3, 1) = 0
++0       <  S   0:0(0)         win 32792  <mss 1000, sackOK, nop, nop, nop, wscale 7, mpcapable v1 flags[flag_h] nokey>
++0       >  S.  0:0(0)  ack 1             <mss 1460, nop, nop, sackOK, nop, wscale 8, mpcapable v1 flags[flag_h] key[skey]>
++0.1     <   .  1:1(0)  ack 1  win 257                                               <mpcapable v1 flags[flag_h] key[ckey=2, skey]>
++0     accept(3, ..., ...) = 4
+
+// shutdown
++0     shutdown(4, SHUT_WR) = 0
++0       >   .  1:1(0)        ack 1                <dss dack4=1 dsn8=1 ssn=0 dll=1 nocs fin, nop, nop>
+
+// ACK the data_fin
++0       <   .  1:1001(1000)  ack 1     win 450    <dss dack8=2 dsn8=1 ssn=0 dll=1001 nocs fin, nop, nop>
++0       >  F.  1:1(0)        ack 1001
++0       <  F.  1001:1001(0)  ack 2     win 450
+
+// Is it normal to ACK the 1000 bytes, but not being able to read them?
```

Comment:
maybe not, but probably fine to ignore this: this should not happen, it is when exchanging the FIN, etc.